content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -50,24 +50,25 @@ The .NET Tracer supports automatic instrumentation on the following architecture
 | Processor architecture                    | Support level         | Package version                        |
 | ------------------------------------------|-----------------------|----------------------------------------|
 | Windows x64 (`win-x64`)                   | [GA](#support-ga)     | latest                                 |
-| Windows x86 (`win-x86`)                   | [GA](#support-ga)     | < 3.0.0 (e.g. 2.56.0)                  |
+| Windows x86 (`win-x86`)                   | [GA](#support-ga)     | latest                                 |
```

Comment:
Do we or do we not support instrumenting `win-x86` processes? (regardless of the OS architecture)

Why did this table say `< 3.0.0`? I think it was getting confused with the fact that the MSI installed no longer works on Windows x86. But OS architecture is not the same as process architecture.

Reply:
Update: We _do_ support instrumenting `win-x86` processes in `dd-trace-dotnet` 3.x